test/addons/argocd/start

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+import json
+import time
+
+from drenv import kubectl
+from drenv import minikube
+from drenv import commands
+from drenv import cache
+
+
+def deploy(cluster):
+    print("Deploying argocd")
+    kubectl.apply("--filename", "argocd-ns.yaml", context=cluster)
+    path = cache.path("addons/argocd-install.yaml")
+    cache.fetch(".", path)
+    kubectl.apply("--filename", path, "--namespace", "argocd", context=cluster)
+
+
+def wait(cluster):
+    print("Waiting until all deployments are available")
+    kubectl.wait(
+        "deploy",
+        "--all",
+        "--for=condition=Available",
+        "--namespace=argocd",
+        "--timeout=300s",
+        context=cluster,
+    )
+
+
+if len(sys.argv) != 4:
+    print(f"Usage: {sys.argv[0]} hubcluster drcluster1 drcluster2")
+    sys.exit(1)
+
+os.chdir(os.path.dirname(__file__))
+cluster = sys.argv[1]
+
+deploy(cluster)
+
+time.sleep(10)
```

Comment:
Why do we need this sleep? Sleeping without checking anything is not a valid way to deploy stuff since it is likely to fail at some point. It this is a temporary hack, please add a comment explaining it.

Reply:
removed it.